FreeRTOS/FreeRTOS
# Feature/multiple direct to task notifications

Description
-----------
Test file and updated Win32 project for the change that replaced the single task notification per task with an array of task notifications per task.  See https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/63

This PR introduced FreeRTOS/Demo/Common/Minimal/TaskNotifyArray.c which tests using different indexes within the new array of task notifications.  The new tests in that file are in addition to, not a replacement for those in FreeRTOS/Demo/Common/Minimal/TaskNotify.c.

The PR also updates the Win32 project such that it builds and runs the the tests in TaskNotifyArray.c.

Test Steps
-----------
Run the Win32 project.  It calls vStartTaskNotifyArrayTask() to create the tests implemented in TaskNotifyArray.c, and periodically calls xAreTaskNotificationArrayTasksStillRunning() form the 'check' task to ensure the new tests are still running.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/63


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
